cms/envs/common.py

```diff
@@ -551,6 +551,17 @@
     #   https://github.com/openedx/openedx-events/issues/265
     # .. toggle_tickets: https://github.com/edx/edx-arch-experiments/issues/381
     'ENABLE_SEND_XBLOCK_LIFECYCLE_EVENTS_OVER_BUS': False,
+
+    # .. toggle_name: FEATURES['ENABLE_HOME_PAGE_COURSE_API_V2']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Enables the new home page course v2 API, which is a new version of the home page course
+    #   API with pagination, filter and ordering capabilities.
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2024-03-14
+    # .. toggle_target_removal_date: None
+    # .. toggle_tickets: https://github.com/openedx/edx-platform/pull/34173
+    'ENABLE_HOME_PAGE_COURSE_API_V2': False,
```

Comment:
So the idea is that we deploy this with default False, then merge the UI changes at the MFE and in the mean time make sure that edx.org turns this on?

Should we make this already the default True and confirm with edx.org that they either take this changes and are ready to turn the V2 off if something doesn't work for them?

Reply:
Yes, I thought whoever wants to use this feature should turn it on manually. But it also sounds good for the feature to be on by default, and it makes sense since it's listed as being released in Redwood. I need help finding the link for that now, could you add it so it's visible? Thanks

Reply:
This is the planning sheet for the releases and the link is direct to the cell for this scope

https://docs.google.com/spreadsheets/d/1tWgp9LXNg4sfWYd_0ghNl6qfIZZ9851MtGBXPeSgzFs/edit#gid=0&range=F69